projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -454,6 +454,8 @@ function get_element_class_names_from_attributes( $attributes ) {
  * @return array
  */
 function get_element_styles_from_attributes( $attributes ) {
+	$is_button_only_style = get_attribute( $attributes, 'className' ) === 'is-style-button';
```

Comment:
I think you can check `$attributes[ 'style' ]` here

Reply:
Nope, it's not there:

```
Array
(
    [className] => is-style-button
    [showSubscribersTotal] =>
    [buttonOnNewLine] =>
    [successMessage] => Success! An email was just sent to confirm your subscription. Please find the email now and click 'Confirm' to start subscribing.
)
```

or another example:
```
Array
(
    [showSubscribersTotal] => 1
    [buttonWidth] => 50%
    [buttonBackgroundColor] => pale-pink
    [textColor] => black
    [borderColor] => vivid-red
    [padding] => 23
    [spacing] => 50
    [style] => Array
        (
            [spacing] => Array
                (
                    [padding] => Array
                        (
                            [right] => var:preset|spacing|30
                            [left] => var:preset|spacing|30
                            [top] => var:preset|spacing|40
                            [bottom] => var:preset|spacing|40
                        )

                    [margin] => Array
                        (
                            [right] => var:preset|spacing|20
                            [left] => var:preset|spacing|20
                        )

                )

        )

    [buttonOnNewLine] =>
    [successMessage] => Success! An email was just sent to confirm your subscription. Please find the email now and click 'Confirm' to start subscribing.
)
```

Reply:
🤔 dreamt about it!